internal/compiler/passes/resolving.rs

```diff
@@ -276,7 +276,7 @@ impl Expression {
         )
     }
 
-    fn from_expression_node(node: syntax_nodes::Expression, ctx: &mut LookupCtx) -> Self {
+    pub fn from_expression_node(node: syntax_nodes::Expression, ctx: &mut LookupCtx) -> Self {
```

Comment:
This innocent looking change is making more API available :-)

So far I used the BindingExpression eval function, which was public anyway, but this one is so much more useful to me as it allows to look as TwoWayBindings in a generic way, too ;-)